### PR TITLE
0.4.7 release preparation

### PR DESCRIPTION
**Pull request by dominikl**
Set version to 0.4.7 ; update Changelog and Readme.
/cc @jburel

### CODE REVIEW

**Comment by dominikl** [comment]
Everything alright, but travis build exits with `1`...

```
...
OK:       160
Failed:   0
Warnings: 0
Skipped:  0
> 
> 
travis_fold:end:lib-test
.omero/docker: line 56: LOGS: unbound variable
travis_time:end:04433b25:start=1561455893122772643,finish=1561457051482357320,duration=1158359584677
[0K[31;1mThe command ".omero/lib-docker" exited with 1.[0m
```

**Comment by joshmoore** [comment]
Sorry, @dominikl. That's my fault in rushing commits this morning in trying to get everything. I'll push yet another commit and relaunch.

**Comment by dominikl** [comment]
No problem. I just had no idea why it suddenly failed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+0.4.7 (June 2019)
+-----------------
+- update to OMERO 5.5
+- preserve backward compatibility to OMERO 5.4
+
 0.4.6 (June 2019)
 -----------------
 - fixed bug in Image.getPixelValues() method which returned
```
Comment (jburel) [inline CHANGELOG.md]: maybe remove usage of past tense for previous release

Comment (dominikl) [reply CHANGELOG.md]: I'll use past tense, too; looks like it's always been past tense for the previous releases.

**Comment by jburel** [comment]
Changes look good. Please do go ahead with release.

**Pull request merged**